pkgs/press/src/constants.ts

```diff
@@ -5,4 +5,4 @@ export const REGION = 'us-west-1'
 export const SENTRY_CLI = resolve(__dirname, '../node_modules/.bin/sentry-cli')
 export const TAG_BRANCH_PREFIX = 'branch-'
 export const TAG_COMMIT_PREFIX = 'commit-'
-export const TAG_VERSION_PREFIX = 'version-'
+export const TAG_VERSION_PREFIX = process.env.TAG_VERSION_PREFIX || 'v'
```

Comment:
do we need `v` prefix ? Check some common patterns - https://hub.docker.com/search?image_filter=official&type=image. There are just plain semver, unless it is necessary to indicate an image specifics.

Reply:
the push.ts script expects and relies on a non empty prefix. it needs some more refactoring, and I'd  rather not invest any more time into this, so I'm going with the least ugly solution under the current constraints